lib/src/validator.dart

```diff
@@ -123,8 +121,8 @@ abstract class Validator {
   /// package, in bytes. This is used to validate that it's not too big to
   /// upload to the server.
   static Future<void> runAll(
-      Entrypoint entrypoint, Future<int> packageSize, Uri serverUrl,
-      {List<String> hints, List<String> warnings, List<String> errors}) {
+      Entrypoint entrypoint, Future<int> packageSize, Uri? serverUrl,
+      {List<String>? hints, List<String>? warnings, List<String>? errors}) {
```

Comment:
I think these should be required.
```suggestion
      {required List<String> hints, required List<String> warnings, required List<String> errors}) {
```

Reply:
done